packages/drivers/routerlicious-driver/src/documentStorageService.ts

```diff
@@ -64,21 +67,51 @@ export class DocumentStorageService implements IDocumentStorageService {
             requestVersion = versions[0];
         }
 
-        const rawTree = await this.manager.getTree(requestVersion.treeId);
+        const rawTree = await PerformanceEvent.timedExecAsync(
+            this.logger,
+            {
+                eventName: "getSnapshotTree",
+                treeId: requestVersion.treeId,
+            },
+            async (event) => {
+                const response = await this.manager.getTree(requestVersion!.treeId);
+                event.end({
+                    size: response.tree.length,
+                });
+                return response;
+            },
+        );
         const tree = buildHierarchy(rawTree, this.blobsShaCache);
 
         this._logTailSha = ".logTail" in tree.trees ? tree.trees[".logTail"].blobs.logTail : undefined;
         return tree;
     }
 
     public async getVersions(versionId: string, count: number): Promise<IVersion[]> {
-        const commits = await this.manager.getCommits(versionId ? versionId : this.id, count);
+        const id = versionId ? versionId : this.id;
+        const commits = await PerformanceEvent.timedExecAsync(
+            this.logger,
+            {
+                eventName: "getVersions",
+                versionId: id,
+                count,
+            },
+            async (event) => {
+                const response = await this.manager.getCommits(id, count);
+                event.end();
```

Comment:
you do not need .end() here if you are not adding any props - it will happen automatically.
So you can do 
```
async (event) => this.manager.getCommits(id, count),
```
and that will suffice.